### PR TITLE
Updated LibMakefile to work with latest fdk-aac library (2.0.0)

### PR DESCRIPTION
**Pull request by asantoni**
I've updated LibMakefile to work with the new 2.0.0 version of the fdk-aac library (https://github.com/mstorsjo/fdk-aac). I did this by merging over the new source variables. I'm not sure if the relative path prefixes (./) are still needed, but I was able to compile using this updated Makefile and mingw without them.

### CODE REVIEW

**Comment by nu774** [comment]
I appreciate your work, but I don't intend to maintain this project ( I consider this to be already dead).

Both of fdkaac and fdk-aac library are using GNU autotools (configure.ac and Makefile.am).
Building them is easy in the Unix-like environment.
You don't need extra something. You just do something like the following in the source tree:
```
$ autoconf -fiv
$ ./configure
$ make
$ sudo make install
```
It's pretty standard way of building software.

I have created hand-crafted makefile (LibMakefile and AppMakefile) just because I wanted to avoid extra dependency on autotools which is not necessary when building for Win32 only.
In other words, If you already have Unix-like environment + MinGW cross compilers, just go the standard way.
You don't need this project to build fdk-aac library and fdkaac.

**Comment by asantoni** [comment]
Got it, thanks. I was trying to cross-compile it for Windows, since there didn't used to be an easy way to build on Windows. However, after your tip, I ended up just using the Visual Studio project that's included now. (I did have to move the "fdk-aac" library source tree inside the fdkaac directory, but otherwise it built perfectly.)
